OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -237,6 +237,11 @@ bool IsBlockedBy(Actor self, Actor otherActor, Actor ignoreActor, CellConditions
 				IsMovingInMyDirection(self, otherActor))
 				return false;
 
+			// If there is a closed gate / temporary blocker in our way and we can't open it, we are blocked.
```

Comment:
Could this be changed to something like `We are not blocked by a temporary blocker that we can remove.` so you don't have to mentally negate the comment to match the code?

(I'm basically just asking this to be consistent with the other code & comments - parts that return false use "we are not blocked" whilst parts that return true use "we are blocked")